### PR TITLE
draft: Ready Player One

### PR DESCRIPTION
**Pull request by badsectortv**
Here's a first go at an MD only article

### CODE REVIEW

File: _posts/2016-07-21-The-Ready-Player-One-Approach-To-Defense.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+layout: post
+title: The Ready Player One Approach to Defense
+date: 21-07-2016
+categories: vanity security thinks sci-fi
+---
+# The Ready Player One Approach to Defense
+
+> What if computer emulation wasn't just about nostalgia? What if we could leverage the power of being olde and having knowledge of defunct hardware and software and legitimately put it to use today?
+
+![Rad!]({{site.url}}assets/images/retro-console.gif)
```
Comment (oholiab) [inline _posts/2016-07-21-The-Ready-Player-One-Approach-To-Defense.md:11]: How do you feel about linking this externally (off of grimmwa.re) until we figure out what we're actually going to do with assets and then shipping it?

We can come back to it later with a long term solution.

Comment (badsectortv) [reply _posts/2016-07-21-The-Ready-Player-One-Approach-To-Defense.md:11]: It's not very portable though is it. I like that the repository is entirely self contained but I also don't like having a jekyll link in the markdown and would prefer to have a global URL. I'm honestly open to suggestion.

Comment (oholiab) [reply _posts/2016-07-21-The-Ready-Player-One-Approach-To-Defense.md:11]: Yeah I agree completely but I think that committing binaries to the repository is a decision that it's difficult to claw back from whereas linking remotely and then updating later is a lot easier to do.

I don't doubt we can automate a git-annex workflow whereby images commited in git-annex are hosted remotely on fuc but the binary data doesn't make its way into the repo so we don't end up versioning a bunch of blobs and making clones slow

Comment (badsectortv) [reply _posts/2016-07-21-The-Ready-Player-One-Approach-To-Defense.md:11]: OK, I'll see about setting up a FUComp subdomain for image hosting tonight and then we'll just have to work out a method of getting images into that easily.

**Pull request closed without merging**